cli/file_copy_getter.go

```diff
@@ -30,7 +31,7 @@ func (g *FileCopyGetter) Get(dst string, u *url.URL) error {
 		return fmt.Errorf("source path must be a directory")
 	}
 
-	return util.CopyFolderContents(path, dst)
+	return util.CopyFolderContents(path, dst, ".terragrunt-source-manifest")
```

Comment:
Shouldn't this be an absolute path, not a relative one? It seems important that the manifest file is stored in the proper folder!

Reply:
Oh, it's not a path, but a name. It's stored in the `dst` folder.